auslib/web/api_validator.py

```diff
@@ -0,0 +1,26 @@
+import flask
+from connexion.decorators.validation import ParameterValidator
+
+
+CONNEXION_IGNORE_VALIDATION_AVAST = ["avast", "force"]
+
+
+class BalrogParameterValidator(ParameterValidator):
+    def __init__(self, *args, **kwargs):
+        super(BalrogParameterValidator, self).__init__(*args, **kwargs)
+
+    def validate_path_parameter(self, args, param):
+        val = args.get(param['name'].replace('-', '_'))
```

Comment:
If a path parameter is not supplied, is required and have a default value, the default value is used.
Like here: https://github.com/mozilla/balrog/blob/master/auslib/web/base.py#L101